content/en/observability_pipelines/monitoring.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Monitoring
+kind: documentation
+disable_toc: false
+---
+
+## Overview
+
+In [Observability Pipelines][1], your pipelines are comprised of components that collect, process, and route your observability data. The health of your pipelines and components are indicated by health statuses and graphs, as well as resource utilization and data delivery graphs.
+
+Health statuses are determined by specific metrics based on thresholds and default time windows. The available statuses are as follows:
+
+- `Healthy`: Indicates the Worker is not falling behind.
+- `Warning`: Indicates the Worker is not performing optimally and is at risk of falling behind. The Worker may fall behind due to issues such as a downstream destination or service causing back pressure to build up and there are not enough resources being provisioned for the Workers.
+- `Critical`: Indicates that the Worker is falling behind. If the Worker is falling behind, it may be at the risk of dropping data; however, so long as your pipelines are [architected][2] and configured correctly, the Worker does not drop data unintentionally.
+
+Internal metrics, which are grouped by health, data delivery, and resource utilization, drives the overall health status of your pipeline and its components.
+
+Health graphs are available for the following metrics:
+- Events unintentionally dropped
+- Errors
+- Lag time (only available for sources)
+- Lag time rate of change (only available for sources)
+- Utilization
+
+Data delivery graphs are available for the following metrics:
+- Events in/out per second
+- Bytes in/out per second
+
+Resource utilization graphs are available for the following metrics:
+- CPU usage
+- Memory usage
+- Disk usage (only available for destinations)
+
+## See the status of your pipelines and components
+
+1. Navigate to [Observability Pipelines][3].
+1. Click on a pipeline.
+1. Hover over the graphs to see specific data points.
+
+## Pipeline resource utilization health metrics
+
+| Metric        | OK       | Warning    | Critical  | Description                       |
+| ------------  | :------: | :--------: | :-------: | --------------------------------- |
+| CPU usage     | <= 0.85  | > 0.85     | N/A       | Tracks how much CPU a Worker process is using. <br> A value of `1` indicates that a Worker process does not have any more headroom in the host or compute units running it. This can lead to possible issues such as processing latency going out of bounds, upstream/downstream overload, and so on.|
```

Comment:
```suggestion
| CPU usage     | <= 0.85  | > 0.85     | N/A       | Tracks how much CPU a Worker process is using. <br><br> A value of `1` indicates that a Worker process does not have any more headroom in the host or compute units running it. This can lead to possible issues such as processing latency going out of bounds, upstream/downstream overload, and so on.|
```